components/layout/LayoutNavBar.vue

```diff
@@ -147,13 +170,63 @@ function closeUserMenu() {
     userDropdownComponent.value.hideMenu();
   }
 }
+
+// Randomly chose from a list of phrases for the tooltip
+const tooltipPhrases = [
+  'Trans Rights are Human Rights',
+  'Protect Trans Kids',
+  'Protect the Dolls',
+  'Trans Rights, Now and Always',
+  'With Love To All Our Trans Siblings',
+  'One Day We Will Win',
+  'Trans Joy is Revolutionary',
+  'You Belong Here',
+  'Trans Lives Matter',
+  'Support Trans Artists',
+  'Trans Visibility Saves Lives',
+  'Trans People Are Loved',
+  'Trans Futures Are Bright',
+  'Trans Liberation For All',
+  'Trans Pride Every Day'
+];
+
+const randomTooltipPhrase = computed(() => {
+  const randomIndex = Math.floor(Math.random() * tooltipPhrases.length);
+  return tooltipPhrases[randomIndex];
+});
+
+const anchor = ref({ x: 0, y: 0 });
+
+const reference = computed(() => ({
+  getBoundingClientRect: () => ({
+    width: 0,
+    height: 0,
+    left: anchor.value.x,
+    right: anchor.value.x,
+    top: anchor.value.y,
+    bottom: anchor.value.y,
+    ...anchor.value
+  })
+}));
 </script>
 
-<style lang="postcss" scoped>
+<style scoped>
+@reference '@/assets/css/main.css';
 nav > a.router-link-exact-active {
   @apply text-sta-orange;
 }
 .auth-button {
   @apply inline-block;
 }
+.transflag {
```

Comment:
Really, this should also be separate in a different commit. If it's a long-term push, this should be one thing, if it's a short term/temporary thing/only for a month/whatever, then an issue should be made to reverse when appropriate, with links to this line of code (and code elsewhere of course!).